explore/src/main/scala/explore/config/ConfigEditState.scala

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.config
+
+import cats.Eq
+
+enum ConfigEditState {
+  case TableView, DetailsView, SimpleEdit, AdvancedEdit
+}
+
+object ConfigEditState {
+  implicit val eqConfigEditState: Eq[ConfigEditState] = Eq.instance {
```

Comment:
We could experiment with `kittens` and this would become 
`enum ConfigEditState derives Eq`

WDYT?

Reply:
I would like to be able to derive `Eq`.  I can pull in `kittens` and try it in another PR.

Reply:
Kittens is still i milestone state thus I'd rather test it on explore before moving it to more core libs

I think it uses macros maybe worth test it the size of js doesn't explode using it